tools/image_processing/imagej2/imagej2_analyze_skeleton/imagej2_analyze_skeleton.xml

```diff
@@ -78,18 +78,28 @@
         </test>
     </tests>
     <help>
+
+.. class:: warningmark
+
+The underlying AnalyzeSkeleton Fiji plugin works on binary images, so other image types will
+automatically be converted to binary before they are analyzed.  This step is performed using the
+ImageJ2 **Make Binary** command with the following settings: **Iterations:** 1, **Count:** 1,
+**Black background:** No, **Pad edges when eroding:** No.  If these settings are not appropriate,
+first manually convert the image to binary using the **Convert to binary (black and white) with
```

Comment:
Hi Greg,

fimo wrapper is here: https://github.com/galaxyproject/galaxy/blob/dev/tools/meme/fimo.xml
I guess it is only deactivated from the tool_conf but not removed from the code base.

I would love to see them as IUC repository, but I guess this would need a tool migration at some point. But this is probably not a reason to not have them in the ToolShed already.

ping @davebx as he is the master of tool migrations :)